### ISSUE TITLE
Oops! Something went wrong - problem

### ISSUE THREAD

**Issue by sfc90** [FriendsOfFlarum/linguist#29]
I install Linguist on fresh new Flarum v. 1.0.1.

I want active and -- Oops! Something went wrong. Please reload the page and try again.
I clear cachce in Flarum and in Composer and nothing helped

**Comment by clarkwinkelmann** [comment]
Please check your Flarum log file (`<flarum>/storage/logs`) and browser javascript console (under development tools) for an error message. You can also enable `debug` mode in `config.php` and try again.

**Comment by rg190** [comment]
Works, thanks !

**Comment by clarkwinkelmann** [comment]
You managed to fix it? Did you figure out the cause or did debug mode just fix it?

Make sure you disable debug mode once you're done with testing. Debug mode just acts as a constant cache clear.

**Comment by rg190** [comment]
Yes i fix it, in config.php I turned on debug .. then I activated the plugin and then I turned off debug again
